src/encoded/types/donor.py

```diff
@@ -12,7 +12,7 @@
 
 class Donor(Item):
     base_types = ['Donor'] + Item.base_types
-    embedded = ['organism']
+    embedded = ['organism', 'references']
```

Comment:
The subtypes of Donor which override embedded need to have this change made too. Taking FlyDonor as an example, it can be cleaned up to first copy Donor's embeds.

Currently:
```python
    embedded = ['organism', 'constructs', 'constructs.target']
```
Should change to:
```python
    embedded = Donor.embedded + ['constructs', 'constructs.target']
```